main.py

```diff
@@ -611,49 +611,50 @@ def get(self):
             self.error(403)
             return
 
-        verification = {}
-
-        query = db.GqlQuery("SELECT * FROM User")
-        for query_result in query:
-            if not query_result.email in verification:
-                verification[query_result.email] = {'verified': 0, 'unverified': 0}
-            verification[query_result.email][self.get_key(query_result.verified)] += 1
+        verification = self.read_db()
 
         html = "<h1>Stats</h1>"
 
         html += "<h2>" + str(len(verification))
         html += " unique emails entered</h2>"
 
-        only_verified = sorted([ k for k,v in verification.iteritems() if v['verified'] == 1 and v['unverified'] == 0 ])
+        only_verified = sorted([ k for k,v in verification.iteritems() if len(v['verified']) == 1 and len(v['unverified']) == 0 ])
         html += "<h3>" + str(len(only_verified)) + " emails in good condition</h3>"
         for email in only_verified:
             html += email + "<br>"
 
-        verified_and_unverified = sorted([ k for k,v in verification.iteritems() if v['verified'] >= 1 and v['unverified'] >= 1 ])
+        verified_and_unverified = sorted([ k for k,v in verification.iteritems() if len(v['verified']) >= 1 and len(v['unverified']) >= 1 ])
         html += "<h3>" + str(len(verified_and_unverified)) + " emails with verified and unverified records</h3>"
         for email in verified_and_unverified:
-            html += email + " [" + str(verification[email]['unverified']) + "]<br>"
+            html += email + " [" + str(len(verification[email]['unverified'])) + "]<br>"
 
-        only_unverified = sorted([ k for k,v in verification.iteritems() if v['verified'] == 0 and v['unverified'] >= 1 ])
+        only_unverified = sorted([ k for k,v in verification.iteritems() if len(v['verified']) == 0 and len(v['unverified']) >= 1 ])
         html += "<h3>" + str(len(only_unverified)) + " emails with only unverified records</h3>"
         for email in only_unverified:
-            html += email + " [" + str(verification[email]['unverified']) + "]<br>"
+            html += email + " [" + str(len(verification[email]['unverified'])) + "]<br>"
```

Comment:
add a thing here to print out if a schedule exists for the specified email (so we know if they even can log in or not)